client-side/src/components/statistics.jsx

```diff
@@ -1,30 +1,37 @@
 import React, { useState, useCallback } from 'react';
 import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client';
-import DateTimePicker from '../stories/datePicker/DatePicker.jsx';
+import DateTimePicker from '../stories/DateTime/DateTimePicker.jsx';
 import VisitedWebsitesComponent from './statistics/graphs.jsx';
-
+import GraphBar from './statistics/graphBar.jsx';
+import RadioButtonComponent from '../stories/RadioButton/radio-Button.jsx';
 const Statistics = () => {
     const [startDate, setStartDate] = useState('');
     const [endDate, setEndDate] = useState('');
-    const [showVisitedWebsites, setShowVisitedWebsites] = useState(false);
+    const [show, setShow] = useState(false);
+    const [whatShow, setWhatShow] = useState('BarChart');

```

Comment:
You shouldn't have 2 states, case of whatShow === undefined could be the same as show ==== false.
rename whatShow to selectedBar or something like that

Reply:
I did it, thanks